src/components/shared/CallNodeContextMenu.js

```diff
@@ -504,6 +500,9 @@ class CallNodeContextMenuImpl extends React.PureComponent<Props> {
     const fileName =
       filePath &&
       parseFileNameFromSymbolication(filePath).path.match(/[^\\/]+$/)?.[0];
+
+    const callNodeTable = callNodeInfo.getNonInvertedCallNodeTable();
```

Comment:
note to self, this change could have been in the previous commit (use the non inverted table for the recursive function transform)